index.html

```diff
@@ -58,9 +69,25 @@
     <script type="text/javascript" src="public/d3.js"></script>
     <script type="text/javascript" src="public/bitemp_graph.js"></script>
     <script type="text/javascript" src="public/bitemp_graph_helper.js"></script>
+    <script type="text/javascript" src="public/input_text_load_data.js"></script>
 
     <div id="bar-chart-large"></div>
-	
+  
+    <script type="text/javascript">
+      $("input[name='insert']").click(function(){
```

Comment:
What is this click handler for? Doesn't appear to correspond to anything in the DOM.